plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel

```diff
@@ -61,7 +61,8 @@ global proc AEmayaUsdProxyShapeTemplate( string $nodeName )
             editorTemplate -addControl "drawProxyPurpose";
             editorTemplate -addControl "drawRenderPurpose";
             //editorTemplate -addControl "primPath";
-            editorTemplate -addControl "excludePrimPaths";
+            editorTemplate -ann `getMayaUsdString("kExcludePrimPathsAnn")`
+                           -addControl "excludePrimPaths";
```

Comment:
As an FYI, I have moved all the content from this file into a "base" version. And then this file simply includes the base. https://github.com/Autodesk/maya-usd/pull/1174
So we should coordinate getting our PRs merged. Whomever goes 2nd will need to resolve the merge conflict.